frontend/src/app/components/new-task-form/new-task-form.component.ts

```diff
@@ -23,4 +23,11 @@ export class NewTaskFormComponent {
     };
     this.tasksService.createTask$(task).subscribe();
   }
+
+  hideTaskForm(): void {
+  //  this.taskFormVisible = false;
+    this.taskFormVisible.emit(false);
```

Comment:
This one `taskFormVisible` will be just `cancel`

Reply:
Done